library/src/main/java/asia/ivity/android/marqueeview/MarqueeView.java

```diff
@@ -42,6 +41,9 @@ public class MarqueeView extends LinearLayout {
 
     private float mTextDifference;
 
+    //Change from virtual width to use device width
```

Comment:
Comment is probably not needed (as it's more explaining why the change was made than what the code is for).

Reply:
Noted.